ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -292,12 +299,25 @@ StatusScrollView {
                                 root.selectedHoldingsModel, ["key", "amount"])
                 }
 
+                function checkIfWeShouldShowNetworkWarning(entry) {
```

Comment:
The function itself only checks if the given chain is part of `enabledChainIds`! ;)
```suggestion
                function isEnabledChain(chain) {
```